app/templates/buyers/studios_start_page.html

```diff
@@ -0,0 +1,99 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}
+Find a user research lab - Digital Marketplace
+{% endblock %}
+
+{% block breadcrumb %}
+    {%
+    with items = [
+        {
+        "link": "/",
+        "label": "Digital Marketplace"
+        }
+    ]
+    %}
+        {% include "toolkit/breadcrumb.html" %}
+    {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="grid-row">
+    <div class="column-two-thirds large-paragraph">
+        {% with
+        smaller = true,
+        heading = "Find a user research lab"
+        %}
+        {% include 'toolkit/page-heading.html' %}
+        {% endwith %}
+    </div>
+
+    <div class="column-two-thirds large-paragraph">
+        <div class="marketplace-paragraph">
+            <p>
+                To find a user research lab, you need to tell suppliers what facilities you need and when. They’ll then tell you what they can do and how much it will cost.
+            </p>
+        </div>
+
+        <h2>Before you start</h2>
+        <ol class="list-number">
+            <li>
+                Prepare your requirements and set evaluation criteria.<br />
+                eg a lab in York with a desktop PC and an observation room
+            </li>
+            <li>Get approval to buy what you need.</li>
+        </ol>
+
+        <h2>Create a shortlist</h2>
+        <ol class="list-number" start="3">
+            <li>Download the list of user research labs from the Digital Marketplace.</li>
+            <li>Filter the list of labs on location and your requirements to create a shortlist.</li>
+        </ol>
+
+        <h2>Contact your shortlist</h2>
+        <ol class="list-number" start="5">
+            <li>Send your requirements and evaluation criteria to shortlisted suppliers.</li>
+            <li>Answer supplier questions.</li>
+            <li>Suppliers will tell you if they can meet your requirements.</li>
+        </ol>
+
+        <h2>Evaluate</h2>
+        <ol class="list-number" start="8">
+            <li>Evaluate suppliers responses on:
+                <div class="explanation-list">
+                    <ul class="list-bullet">
+                        <li>availability</li>
+                        <li>location, facilities and accessibility</li>
+                        <li>price</li>
+                    </ul>
+                </div>
+            </li>
+            <li>Award a contract to the supplier that best meets your needs.</li>
+        </ol>
+
+        <div class="marketplace-paragraph">
+            <p>Read more about <a href="https://www.gov.uk/guidance/how-to-hire-user-research-labs-on-the-digital-marketplace">how to hire user research labs</a>.</p>
+        </div>
+        
+        <h2>Download the list of labs</h2>
+
+        {%
+        with
+            items = [
+            {
+            "title": "List of labs (???K)",
```

Comment:
Let's just remove the size for now. We can add it if we have time to do a release once we have the final files.